src/components/interfaces/MOBILE_API.xml

```diff
@@ -5001,6 +5040,38 @@
     </param>
   </function>
 
+  <function name="GetSystemCapability" functionID="GetSystemCapabilityID" messagetype="request">
+    <param name="systemCapabilityType" type="SystemCapabilityType" mandatory="true">
+      <description>The type of system capability to get more information on</description>
+    </param>
+  </function>
+
+  <function name="GetSystemCapability" functionID="GetSystemCapabilityID" messagetype="response">
+      <param name="systemCapability" type="SystemCapability" mandatory="true">    
+      </param>
+      <param name="resultCode" type="Result" platform="documentation" mandatory="true">
+          <description>See Result</description>
+          <element name="SUCCESS"/>
+          <element name="INVALID_DATA"/>
+          <element name="OUT_OF_MEMORY"/>
+          <element name="TOO_MANY_PENDING_REQUESTS"/>
+          <element name="APPLICATION_NOT_REGISTERED"/>
+          <element name="GENERIC_ERROR"/>
+          <element name="REJECTED"/>
+          <element name="IGNORED"/>
+          <element name="DISALLOWED"/>
+          <element name="USER_DISALLOWED"/>
+          <element name="UNSUPPORTED_RESOURCE"/><description>The capability does not exist on the module</description>
+          <element name="DATA_NOT_AVAILABLE"/><description>The capability should exist on the module but there was an error retrieving the data.</description>

```

Comment:
I know it's done elsewhere, but the descriptions should be inside the `element` tag, meaning it should not be standalone. Joey did this in the [rpc_spec repo](https://github.com/smartdevicelink/rpc_spec/commit/45eec48de8a929c79f11b325eba7eaf467c03e0d).